addons/website_links/static/src/js/website_links.js

```diff
@@ -533,6 +535,26 @@ publicWidget.registry.websiteLinks = publicWidget.Widget.extend({
     },
 });
 
+popupWidget.include({
+    /**
+     * @override
+     */
+    _onHideModal: function (){
+        this._super.apply(this, arguments);
+        const params = new URLSearchParams(window.location.search);
+        const trackingFields = {
+            'utm_campaign': 'odoo_utm_campaign',
+            'utm_source': 'odoo_utm_source',
+            'utm_medium': 'odoo_utm_medium',
+        };
+        for (const [key, value] of params) {
+            if (key in trackingFields) {
+                setCookie(trackingFields[key], value, 31 * 24 * 60 * 60, 'required');
```

Comment:
Comment what nle said, otherwise no one will have a clue that it's to mimick the python forced date for utm cookies